### PR TITLE
docs(kuma-cp) Fault Injection

### PR DESCRIPTION
**Pull request by lobkovilya**
Add documentation for the Fault Injection policy. 
Also, I incremented the version up to  0.4.1, not sure that it is needed.

### CODE REVIEW

**Comment by subnetmarco** [comment]
@lobkovilya please do not create a new 0.4.1 version but add the content in the 0.4.0 folder (and open against `next` like you have done). Prior to the release we will take the `0.4.0` in `next` and convert it to `0.4.1`.

File: docs/docs/0.4.0/documentation/http-api.md [hunk 1]
```diff
@@ -1318,6 +1318,164 @@ Example:
...
+  {
+   "match": {
+    "protocol": "http",
+    "service": "backend"
+   }
+  }
+ ],
+ "conf": {
+  "delay": {
+   "percentage": 50,
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/documentation/http-api.md:1356]: Can we pick different value like 50.5 so we show that this is double and not int?

File: docs/docs/0.4.0/documentation/http-api.md [hunk 2]
```diff
@@ -1318,6 +1318,164 @@ Example:
...
+```bash
+curl http://localhost:5681/meshes/default/fault-injections
+```
+```json
+{
+ "items": [
+  {
+   "type": "FaultInjection",
+   "mesh": "default",
+   "name": "fi1.default",
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/documentation/http-api.md:1432]: how about just `fi1`?

File: docs/docs/0.4.0/policies/fault-injection.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        delay:
+            percentage: 50
+            value: 5s
+        responseBandwidth:
+            limit: 50 mbps
+            percentage: 50 
+```
+
+### Sources & Destinations
+`FaultInjection` is a Connection policy, which means it is applied to the connection between dataplanes. As most of the policies, `FaultInjection` supports the powerful mechanism of matching, which allows you to precisely match source and destination services.
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/policies/fault-injection.md:66]: I don't think we use `Connection policy` concept in the docs, so maybe just:
`FaultInjection` is a policy which is applied to the connection between selected dataplanes
?

`source and destination services`
maybe source and destination dataplanes? You can do `service: '*'` and match for other tags

File: docs/docs/0.4.0/policies/fault-injection.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+::: warning
+Since `FaultInjection` policy available only for L7 HTTP traffic, `protocol: http` is mandatory tag both for source and destination selector.
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/policies/fault-injection.md:69]: The warning at the top tells me the same message. Can we merge those and have only this one?

File: docs/docs/0.4.0/policies/fault-injection.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+### HTTP Faults
+
+At least one of the following Faults should be specified.
+#### Abort
+
+Abort defines a configuration of not delivering requests to destination service and replacing the responses from destination dataplane by
+predefined status code.
+
+- httpStatus -  HTTP status code which will be returned to source side
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/policies/fault-injection.md:80]: httpStatus -> `httpStatus` and same with other fields

**Comment by jakubdyszkiewicz** [comment]
Is it added in the policies tab? The one with squares and icons? I think we will need an icon for it. Can you add a card for it?

File: docs/docs/0.4.0/policies/fault-injection.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+On Universal:
+
+```yaml
+type: FaultInjection
+mesh: default
+name: fi1
+sources:
+    - match:
+        service: frontend
+        version: "0.1"
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/policies/fault-injection.md:15]: We've got required http tag, right? We should add it to examples, otherwise they are not valid.

Comment (lobkovilya) [reply docs/docs/0.4.0/policies/fault-injection.md:15]: Sure, thank you! I missed a couple of places

File: docs/docs/0.4.0/documentation/http-api.md [hunk 7]
```diff
@@ -1339,7 +1339,7 @@ curl http://localhost:5681/meshes/default/fault-injections/fi1
    "match": {
     "protocol": "http",
     "service": "frontend",
-    "version": "0.1"
+    "version": "0.1",
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/documentation/http-api.md:1342]: I don't think this is a valid JSON, also check other places here

Comment (lobkovilya) [reply docs/docs/0.4.0/documentation/http-api.md]: Yep, thanks! A bit of Go deformation

**Comment by subnetmarco** [comment]
@bloqhead to add the logo.

**Comment by bloqhead** [comment]
The Policies page is updated for when this is ready to be merged in.

**Comment by jakubdyszkiewicz** [comment]
merging

**Pull request merged**